docs/cloud-tutorials/best-practices.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 8
+title: Best practices
+description: DBOS Cloud Tips
+---
+
+A list of tips for using DBOS Cloud effectively.
+
+## Managing applications
+* Only one request at a time can manipulate an application and you will get an `application is busy` error if you attempt multiple concurrent modifications.
```

Comment:
We can provide a solution here, for example "If you see an `application is busy` error, try to wait for a couple of minutes before retrying".

Reply:
it is hard to provide such a time frame because it dpends on what action is currently being done on the app. Deploys are 30seconds. Deletes a few seconds.